doc/api/next_api_changes/deprecations.rst

```diff
@@ -135,3 +135,11 @@ access the transform classes from the :mod:`.scale` module.
 ``TexManager.cachedir``
 ~~~~~~~~~~~~~~~~~~~~~~~
 Use `matplotlib.get_cachedir()` instead.
+
+Setting `.Line2D`\'s pickradius via `.Line2D.set_picker`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Setting a `.Line2D`\'s pickradius (i.e. the tolerance for pick events and
+containment checks) is deprecated.  Use `.Line2D.set_pickradius` instead.
```

Comment:
This sentence reads funny.  Suggest "Setting a `.Line2D`\'s pickradius via `.Line2D.set_picker`."  Yes, I know it says that the previous line , but if you don't want to repeat, don't repeat any of it?

Reply:
nah, that was just a plain typo, edited to yours.